### DISCUSSION TITLE
Network auto configuration failed during installation.

### DISCUSSION THREAD

**Discussion by Tropical-Smootie** [utmapp/UTM#5318] [Linux]
**Describe the issue**  
during the installation process, I got an error saying "Network auto configuration failed"

Im wondering if it because im using ethernet instead of wifi.

**Configuration**  
* UTM Version: Version 4.2.5 (81)
* macOS Version: ventura 13.3.1 
*  Mac Chip: M2 

![Screenshot 2023-05-10 at 11 03 21 PM](https://github.com/utmapp/UTM/assets/67304179/f2b84862-ee3f-4cfd-9336-6d089311ea31)

**Comment by hanswurst2023** [comment]
I ran into the same issue. What did the trick for me was to go to "Edit selected VM" - Devices/Network and set Network mode to Bridge.

Configuration

    UTM Version: Version 4.2.5 (81)
    macOS Version: ventura 13.3.1(a)
    Mac Chip: M2

**Reply by tgratton** [reply]
Just had this issue, turning off Nord VPN solved it for me.
